scootapi/scoot.thrift

```diff
@@ -29,14 +29,14 @@ enum JobType {
 }
 
 enum RunStatusState {
-  UNKNOWN = 0      # Reserved.
-  PENDING = 1      # Run scheduled but not yet started.
-  RUNNING = 2      # Run is happening.
-  COMPLETE = 3     # Succeeded or failed yielding an exit code. Only state with an exit code.
-  FAILED = 4       # Run mechanism failed in an expected way and is no longer running.
-  ABORTED = 5      # User requested that the run be killed, or task preempted by Scheduler.
-  TIMEDOUT = 6     # Run timed out and was killed.
-  BADREQUEST = 7   # Request rejected due to unexpected failure.
+  UNKNOWN = 0      # Catch-all for indeterminate RunStatusStates. Also an "end state"
+  PENDING = 1      # Waiting to run
+  RUNNING = 2      # Running
+  COMPLETE = 3     # Succeeded or failed yielding an exit code
+  FAILED = 4       # Run mechanism failed and is no longer running
+  ABORTED = 5      # User requested that the run be killed, or task preempted by Scheduler
+  TIMEDOUT = 6     # Run timed out and was killed
+  #BADREQUEST = 7  # DEPRECATED
```

Comment:
If we're deleting code level compatibility for this everywhere else, prefer we just delete it from here

Reply:
👍